pyt/__main__.py

```diff
@@ -30,7 +30,27 @@
 )
 
 
-def main(command_line_args=sys.argv[1:]):  # noqa: C901
+def discover_files(targets, excluded_files, recursive=False):
+    included_files = list()
+    excluded_list = excluded_files.split(",")
+    
+    
+    for target in targets:
+        if os.path.isdir(target):
+                for root, dirs, files in os.walk(target):
```

Comment:
I think this line, `for root, dirs, files in os.walk(target):` is indented one more level than it has to be.

Reply:
i will try to do better for returning "included_files"